amz_review_scraper/amzscraper.py

```diff
@@ -8,59 +8,30 @@
 from config import proxies
```

Comment:
You are definitely making some progress on this class, but one thing I would encourage is separating out the committing to the DB, from the scraping.

Single-Responsibilty-Prinicpal: One method is for scraping, one for committing.
And it will make this more readable and testable!

Reply:
Yeah I was planning to add a class method to the models for storing to DB